dspace-server-webapp/src/main/java/org/dspace/app/rest/repository/RequestItemRepository.java

```diff
@@ -247,11 +247,15 @@ public RequestItemRest put(Context context, HttpServletRequest request,
             message = responseMessageNode.asText();
         }
 
+        JsonNode responseSubjectNode = requestBody.findValue("subject");
+        String subject = null;
+        if (responseSubjectNode != null && !responseSubjectNode.isNull()) {
```

Comment:
I used both `!= null` and `isNull()` because you did so in line 246, @tdonohue. But I wonder why both would be necessary.

Reply:
@MW3000 : It's necessary to fix this behavioral bug https://github.com/DSpace/DSpace/pull/9136